src/test/java/io/confluent/connect/jdbc/sink/JdbcSinkTaskTest.java

```diff
@@ -474,7 +477,40 @@ void initWriter() {
     task.put(records);
     verifyAll();
   }
+  @Test
+  public void testGetAllMessagesExceptionWithoutTrim() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+    JdbcSinkTask task = new JdbcSinkTask();
+    SQLException exception = new BatchUpdateException("Batch entry 0 INSERT INTO \"abc\" (\"c1\",\"c2\",\"c3\",\"c4\") " +
+            "VALUES ('1','2','3',NULL) was aborted: ERROR: null value in column \"c4\" violates not-null constraint\n" +
+            "  Detail: Failing row contains (1, 2, 3, null).  Call getNextException to see other errors in the batch.",
+            new int[0]);
+    String exceptedExceptionMessage = "Exception chain:" + System.lineSeparator() + exception + System.lineSeparator();
+    Method privateMethod = JdbcSinkTask.class.getDeclaredMethod("getAllMessagesException", SQLException.class);
+    task.shouldTrimSensitiveLogs = false;
+    privateMethod.setAccessible(true);
```

Comment:
@b-goyal - typically we prefer using mocks to test the exceptions. I would prefer we don't do reflections. Can you re-work the tests? I am gonna approve and merge to pull the feature in for now

Reply:
Mockito does not provide ways to test private methods, that's why I used reflections here.

Reply:
Not using private method any more: https://github.com/confluentinc/kafka-connect-jdbc/pull/1299